readme.md

```diff
@@ -1,4 +1,4 @@
-# angular2-polyfill [![Build Status](https://travis-ci.org/SamVerschueren/angular2-polyfill.svg?branch=master)](https://travis-ci.org/SamVerschueren/angular2-polyfill)
+# angular2-polyfill [![Build Status](https://travis-ci.org/SamVerschueren/angular2-polyfill.svg?branch=master)](https://travis-ci.org/SamVerschueren/angular2-polyfill)[![CDNJS](https://img.shields.io/cdnjs/v/angular2-polyfill.svg)](https://cdnjs.com/libraries/angular2-polyfill)
```

Comment:
Could you add a newline after the title? When I show multiple badges in my projects, I always show them on a separate line. Thanks!